tools/bpf2c/bpf_code_generator.cpp

```diff
@@ -892,15 +893,13 @@ bpf_code_generator::encode_instructions(const bpf_code_generator::unsafe_string&
                 break;
             case AluOperations::Div:
                 if (is64bit) {
-                    output.lines.push_back(
-                        std::format("{} = {} ? ({} / {}) : 0;", destination, source, destination, source));
+                    type = (inst.offset == 1) ? "(int64_t)" : "";
```

Comment:
Can we add a test to https://github.com/microsoft/ebpf-for-windows/tree/main/tests/bpf2c_tests/expected? 

I.e. create an ELF file with this new instruction, add it to tests\sample and generate the expected bpf2c files?

Reply:
To me that seems redundant with bpf_conformance and not as good since it'd be just one case instead of all the test cases that bpf_conformance has.  I think a much better approach is to snap to the latest version of bpf_conformance and not have redundant tests under tests\sample just to test bpf2c.

Reply:
Sure, that makes sense. I will go create a release in bpf_conformance and a PR in this repo to pick up the latest version.